engine/flink/executor/src/main/scala/pl/touk/nussknacker/engine/process/typeinformation/internal/ContextTypeHelpers.scala

```diff
@@ -15,7 +15,13 @@ object ContextTypeHelpers {
     ConcreteCaseClassTypeInfo(
       ("id", contextIdInfo),
       ("variables", variables),
-      ("parentContext", parentCtx)
+      ("parentContext", parentCtx),
+      (
+        "traceId",
+        new OptionTypeInfo[String, Option[String]](
+          TypeInformation.of(classOf[String])
+        )
```

Comment:
It would be nice to move this to some private def / val, simillar to `contextIdInfo`